frontend/src/interfaces/IDocumentResults.ts

```diff
@@ -0,0 +1,9 @@
+export interface IDocumentFilter {
+  documentType?: number;
```

Comment:
Looks like this should be documentTypeId

Reply:
Done